docker/fd2-up.bash

```diff
@@ -270,9 +290,9 @@ docker rm fd2_mariadb &> /dev/null
 docker rm fd2_phpmyadmin &> /dev/null
 docker rm fd2_farmdata2 &> /dev/null
 
-echo "Starting containers..."
+echo "Starting containers with profiles: $profiles"
 # Note: Any command line args are passed to the docker-compose up command
-docker compose --profile $PROFILE up -d "$@"
+docker compose --profile $profiles up -d "$@"
```

Comment:
Something is going on on this line because I can't run the `./fd2-up.bash` command. No containers are actually brought online. 
![image](https://github.com/DickinsonCollege/FarmData2/assets/31524934/f507097b-5432-4f9a-a18b-5de98c0c479e)